src/model_signing/_hashing/file.py

```diff
@@ -64,8 +51,6 @@ class FileHasher(hashing.HashEngine):
     hashing files, instead of any `HashEngine` instance).
     """
 
-    pass
-
```

Comment:
is this just style?

Reply:
Yeah, we don't need a `pass` if we already have a docstring.